internal/recruiter/repository.go

```diff
@@ -84,7 +84,6 @@ func (r *Repository) SaveRecruiterProfile(dev Recruiter) error {
 		dev.Email,
 		dev.Name,
 		dev.CompanyURL,
-		dev.Slug,
```

Comment:
we might want to keep this for the moment - i will open another item to remove unused fields in the recruiters table